crates/telio-model/src/api_config.rs

```diff
@@ -222,6 +222,47 @@ where
     Ok(Some(eps))
 }
 
+/// Telio task monitor configuration
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize)]
+pub struct FeatureTaskMonitor {
+    /// The number of consecutive wait iterations that should trigger busy loop detection.
+    /// This typically occurs when erroneous code results in a wait loop without any awaits.
+    #[serde(default = "FeatureTaskMonitor::default_max_polls_immediately")]
+    pub max_polls_immediately_ms: Option<u64>,
+
+    /// The frequency at which the same event can be reported by the task monitor, in milliseconds.
+    #[serde(default = "FeatureTaskMonitor::default_duration_between_reporting_same_event")]
+    pub duration_between_reporting_same_event_ms: Option<u64>,
+}
+
+impl FeatureTaskMonitor {
```

Comment:
I my opinion the constants should either be removed or made private. The functions should be enough

Reply:
I need non optional values for place I use them, initialy i designed so that it would just have a default value, but then you have an issue then python test's pass a null. We can probably just fix python and say that they are not allowed to be set to null.

But I will create an improvement task for api_config.rs in general